reading.Rmd

```diff
@@ -1066,6 +1082,10 @@ population_nodes <- html_nodes(page, selectors)
 head(population_nodes)
 ```
 
+```{r echo = FALSE}
+print_html_nodes(head(population_nodes))
```

Comment:
This did not work in the HTML version -- attached is a screenshot of what this output

![scren](https://user-images.githubusercontent.com/5981307/148695511-2d242ec1-8ee1-4425-84db-11ee6846fb11.png)
s

Reply:
checking PDF now...

Reply:
Seems to work fine in PDF. Not sure why there's a difference!